src/algebra/floor.lean

```diff
@@ -271,72 +273,74 @@ variables {a : α} {n : ℕ}
 It is equal to `⌊q⌋` when `q ≥ 0`, and is `0` otherwise. -/
```

Comment:
Similarly with this one, `` `nat_floor x` (notation `⌊x⌋₊`) `` or similar would be good on the line above this one.